vng_api_common/authorizations/config/models.py

```diff
@@ -0,0 +1,39 @@
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+from django.db import models
+from django.utils.module_loading import import_string
+from django.utils.translation import ugettext_lazy as _
+
+from solo.models import SingletonModel
+from zds_client import Client
+
+
+class AuthorizationsConfig(SingletonModel):
+    api_root = models.URLField(_("api root"), unique=True, default='https://ref.tst.vng.cloud/ac/api/v1')
+    component = models.CharField(_("component"), max_length=50, default='ZRC')
+
+    class Meta:
+        verbose_name = _("Autorisatiecomponentconfiguratie")
+
+    def __str__(self):
+        return self.api_root
+
+    @classmethod
+    def get_client(cls) -> Client:
```

Comment:
Looks like we could do with an abstract base model class to get rid of this code duplication between NRC/AC config

Reply:
> The only con - default value of api_root can't be set now.

Sure it can, see: https://docs.djangoproject.com/en/1.11/ref/models/fields/#default
Such a function might just return a `settings.SOMETHING` that is defined in `vng_api_common.conf.api`

Reply:
My bad, you still need to distinguish between the two... `__init__`?

Reply:
made `__init__` methods in successors